### PR TITLE
GsaGH-320 bake aware params

### PR DESCRIPTION
**Pull request by kpne**

### CODE REVIEW

File: GsaGH/Parameters/2_Geometry/GsaElement1dParameter.cs [hunk 1]
```diff
@@ -32,5 +40,16 @@ public override string InstanceDescription
       this.AddRuntimeError($"Data conversion failed from {data.GetTypeName()} to Element1d");
       return new GsaElement1dGoo(null);
     }
+
+    public bool IsBakeCapable => !m_data.IsEmpty;
```
Comment (tlmnrnhrdt) [inline GsaGH/Parameters/2_Geometry/GsaElement1dParameter.cs:44]: that´s a field and doesn´t belong here ;)

File: GsaGH/Parameters/2_Geometry/GsaNode.cs [hunk 2]
```diff
@@ -102,6 +102,7 @@ public bool IsSupport
         UpdatePreview();
       }
     }
+    internal GH_Point GH_Point { get; set; }
```
Comment (tlmnrnhrdt) [inline GsaGH/Parameters/2_Geometry/GsaNode.cs:105]: `internal GH_Point _point`

**Review by tlmnrnhrdt — CHANGES_REQUESTED** [review]
also why all those underscores now?

File: GsaGH/Parameters/2_Geometry/GsaNode.cs (hunk 2 above)
Comment (kpne) [reply GsaGH/Parameters/2_Geometry/GsaNode.cs]: reverted this

File: GsaGH/Parameters/2_Geometry/GsaElement1dParameter.cs (hunk 1 above)
Comment (kpne) [reply GsaGH/Parameters/2_Geometry/GsaElement1dParameter.cs]: moved this - the underscores are from Grasshopper by inheriting stuff from `GH_Goo`

**Review by tlmnrnhrdt — APPROVED** [review]

**Pull request merged**